incubator/piwik/values.yaml

```diff
@@ -0,0 +1,87 @@
+## Bitnami piwik image version
+## ref: https://hub.docker.com/r/bitnami/piwik/tags/
+##
+image: bitnami/piwik:3.0.1-r1
+
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+imagePullPolicy: IfNotPresent
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-piwik#environment-variables
+##
+piwikUsername: user
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://github.com/bitnami/bitnami-docker-piwik#environment-variables
+# piwikPassword:
+
+## Admin email
+## ref: https://github.com/bitnami/bitnami-docker-piwik#environment-variables
+##
+piwikEmail: user@example.com
+
+## Piwik website name
+## Default name website to track in Piwik
+## ref: https://github.com/bitnami/bitnami-docker-piwik#environment-variables
+piwikWebsiteName: example
+
+## Piwik website host
+## Default host website to track in Piwik
+## ref: https://github.com/bitnami/bitnami-docker-piwik#environment-variables
+piwikWebsiteHost: https://example.org
+
+## SMTP mail delivery configuration
+## ref: https://github.com/bitnami/bitnami-docker-piwik#smtp-configuration
+##
+# smtpHost:
+# smtpPort:
+# smtpUser:
+# smtpPassword:
+# smtpProtocol:
+
+##
+## MariaDB chart configuration
+##
+mariadb:
+  ## MariaDB admin password
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#setting-the-root-password-on-first-run
+  ## mariadbRootPassword: secretpassword
+
+  ## Enable persistence using Persistent Volume Claims
+  ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  persistence:
+    enabled: true
+    storageClass: generic
+    accessMode: ReadWriteOnce
+    size: 8Gi
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##
+serviceType: LoadBalancer
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistence:
+  enabled: true
+  apache:
+    storageClass: generic
```

Comment:
this should be commented out after the storage class annotation changes like https://github.com/kubernetes/charts/blob/master/stable/osclass/values.yaml#L88-L91